src/Language/Haskell/Ghcid/Parser.hs

```diff
@@ -64,7 +64,7 @@ parseLoad (map Esc -> xs) = nubOrd $ f xs
 
         -- <no location info>: error:
         f (x:xs)
-            | unescapeE x == "<no location info>: error:"
+            | "<no location info>: error:" `isPrefixOfE` x
```

Comment:
Should we still remove escape characters here?

```suggestion
            | "<no location info>: error:" `isPrefixOfE` unescapeE x
```

Reply:
Nah, `unescapeE` is an `Esc -> String` transform, `isPrefixOfE` handles the `Esc` escaping already.